### DISCUSSION TITLE
Is there a sample config for training the target matcher available?

### DISCUSSION THREAD

**Discussion by bhartm3** [medspacy/medspacy#191] [Q&A] [answered]
Hello,
is there an example config to train the `medspacy_target_matcher` component with the `spacy train` command with some sample data? 
Cheers,
Ben

**Comment by burgersmoke** [comment, marked as answer]
@bhartm3 The answer to this is a bit more involved, but as of this moment, medspacy and its target matcher do not have any machine learned concepts or entities, whether for training or inference.  While spacy itself has some methods of doing this, medspacy does not currently release any such capability.  This discussion topic and my response might provide some additional clarity on this:

https://github.com/medspacy/medspacy/discussions/194
